candi/bashible/common-steps/all/002_apt_timeout.sh.tpl

```diff
@@ -12,8 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{{- if eq .runType "ImageBuilding" }}
+function set_apt_timeout() {
+  if [[ -f /etc/apt/apt.conf.d/99timeout ]]; then
+    return 0
+  fi
+
+  echo 'Acquire::http::Timeout "120";' > /etc/apt/apt.conf.d/99timeout
+}
+
+case $(bb-is-bundle) in
+  debian|ubuntu-lts|astra|altlinux) set_apt_timeout ;;
+esac
```

Comment:
May be simplify this step ?
Something like
if [[ -d /etc/apt/apt.conf.d/ ]]; then do work

Reply:
done